pkgs/development/tools/cdecrypt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "cdecrypt";
+  version = "4.8";
+
+  src = fetchFromGitHub {
+    owner = "VitaSmith";
+    repo = "cdecrypt";
+    rev = "v${version}";
```

To avoid tag/branch confusion.

```suggestion
    rev = "refs/tags/v${version}";
```

---

change applied in rebased commit.